### ISSUE TITLE
How to use on homepage of docs.rs uses deprecated functions

### ISSUE THREAD

**Issue by djeedai** [Lokathor/rusty-xinput#25]
The docs point to deprecated functions like `dynamic_load_xinput()`. There's no mention of what I believe is the proper way, via `XInputHandle.load_default()` _etc._.

https://docs.rs/rusty-xinput/latest/rusty_xinput/

**Comment by Lokathor** [comment]
Good point. If you wanna do a docs PR I'll merge it, otherwise I'll get to this "eventually". The main consumer of the crate at this point is `gilrs`, and they know what they're doing despite any old docs.
